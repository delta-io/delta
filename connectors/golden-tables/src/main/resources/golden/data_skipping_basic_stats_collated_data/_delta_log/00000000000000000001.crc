{"txnId":"387f5bf0-a194-4817-8bc8-9f933aa5d395","tableSizeBytes":710,"numFiles":1,"numDeletedRecordsOpt":0,"numDeletionVectorsOpt":0,"numMetadata":1,"numProtocol":1,"setTransactions":[],"domainMetadata":[],"metadata":{"id":"f1b00907-6fed-498a-8f44-d8c242dd4407","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"c1\",\"type\":\"string\",\"nullable\":true,\"metadata\":{\"__COLLATIONS\":{\"c1\":\"icu.UNICODE_CI\"}}},{\"name\":\"c2\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{"delta.enableDeletionVectors":"true"},"createdTime":1727103099461},"protocol":{"minReaderVersion":3,"minWriterVersion":7,"readerFeatures":["deletionVectors"],"writerFeatures":["deletionVectors","collations-dev","appendOnly","invariants"]},"histogramOpt":{"sortedBinBoundaries":[0,8192,16384,32768,65536,131072,262144,524288,1048576,2097152,4194304,8388608,12582912,16777216,20971520,25165824,29360128,33554432,37748736,41943040,50331648,58720256,67108864,75497472,83886080,92274688,100663296,109051904,117440512,125829120,130023424,134217728,138412032,142606336,146800640,150994944,167772160,184549376,201326592,218103808,234881024,251658240,268435456,285212672,301989888,318767104,335544320,352321536,369098752,385875968,402653184,419430400,436207616,452984832,469762048,486539264,503316480,520093696,536870912,553648128,570425344,587202560,603979776,671088640,738197504,805306368,872415232,939524096,1006632960,1073741824,1140850688,1207959552,1275068416,1342177280,1409286144,1476395008,1610612736,1744830464,1879048192,2013265920,2147483648,2415919104,2684354560,2952790016,3221225472,3489660928,3758096384,4026531840,4294967296,8589934592,17179869184,34359738368,68719476736,137438953472,274877906944],"fileCounts":[1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"totalBytes":[710,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]},"deletedRecordCountsHistogramOpt":{"deletedRecordCounts":[1,0,0,0,0,0,0,0,0,0]},"allFiles":[{"path":"test%25file%25prefix-part-00000-baefcf91-5c61-4964-a42e-958aed50f989-c000.snappy.parquet","partitionValues":{},"size":710,"modificationTime":1727103103361,"dataChange":false,"stats":"{\"numRecords\":3,\"minValues\":{\"c1\":\"a\",\"c2\":\"1\"},\"maxValues\":{\"c1\":\"c\",\"c2\":\"3\"},\"nullCount\":{\"c1\":0,\"c2\":0},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{\"c1\":\"a\"},\"maxValues\":{\"c1\":\"c\"}}}}","tags":{"INSERTION_TIME":"1727103103361000","MIN_INSERTION_TIME":"1727103103361000","MAX_INSERTION_TIME":"1727103103361000","OPTIMIZE_TARGET_SIZE":"268435456"}}]}
