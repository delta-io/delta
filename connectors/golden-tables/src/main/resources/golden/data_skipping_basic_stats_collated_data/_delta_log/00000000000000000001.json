{"commitInfo":{"timestamp":1727103103721,"operation":"WRITE","operationParameters":{"mode":"Append","statsOnLoad":false,"partitionBy":"[]"},"readVersion":0,"isolationLevel":"WriteSerializable","isBlindAppend":true,"operationMetrics":{"numFiles":"1","numOutputRows":"3","numOutputBytes":"710"},"tags":{"noRowsCopied":"true","restoresDeletedRows":"false"},"engineInfo":"Databricks-Runtime/<unknown>","txnId":"387f5bf0-a194-4817-8bc8-9f933aa5d395"}}
{"add":{"path":"test%25file%25prefix-part-00000-baefcf91-5c61-4964-a42e-958aed50f989-c000.snappy.parquet","partitionValues":{},"size":710,"modificationTime":1727103103361,"dataChange":true,"stats":"{\"numRecords\":3,\"minValues\":{\"c1\":\"a\",\"c2\":\"1\"},\"maxValues\":{\"c1\":\"c\",\"c2\":\"3\"},\"nullCount\":{\"c1\":0,\"c2\":0},\"tightBounds\":true,\"statsWithCollation\":{\"icu.UNICODE_CI.153.120.0.0\":{\"minValues\":{\"c1\":\"a\"},\"maxValues\":{\"c1\":\"c\"}}}}","tags":{"INSERTION_TIME":"1727103103361000","MIN_INSERTION_TIME":"1727103103361000","MAX_INSERTION_TIME":"1727103103361000","OPTIMIZE_TARGET_SIZE":"268435456"}}}
